health/micro-ui/web/packages/modules/campaign-manager/src/utils/setupCampaignValidators.js

```
@@ -365,6 +392,75 @@ export const  handleValidate = ({formData,t,setShowToast,hierarchyDefinition,low
         return false;
       }
 
+      // Validate that all attribute fields are filled
+      let attributeFieldsEmpty = false;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)